anomalib/models/dfkde/torch_model.py

```diff
@@ -59,7 +60,8 @@ def __init__(
         self.threshold_offset = threshold_offset
 
         _backbone = getattr(torchvision.models, backbone)
-        self.feature_extractor = FeatureExtractor(backbone=_backbone(pretrained=pre_trained), layers=["avgpool"]).eval()
+        self.layers = layers
```

Comment:
is `self.layers` used? If not maybe we could remove it ?

Reply:
removed in ed11e72